components/subHeader/subHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import styles from "./subHeader.module.css";
+
+export type SubHeaderProps = {
+  infoOne: string;
+  infoOneNumber: string;
```

Comment:
I'd suggest to use type number:
```suggestion
  infoOneNumber: number;
```